static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -354,10 +354,16 @@ function WidgetBuilder({
                 <DataSetChoices
                   label="dataSet"
                   value={state.dataSet}
-                  choices={
-                    state.displayType === DisplayType.TABLE
-                      ? DATASET_CHOICES
-                      : [DATASET_CHOICES[0]]
+                  choices={DATASET_CHOICES}
+                  disabledChoices={
+                    state.displayType !== DisplayType.TABLE
+                      ? [
+                          [
+                            DATASET_CHOICES[1][0],
```

Comment:
Not a blocker, but I wish we had this as named datasets so that it does not break after we change the order of things.

Reply:
sure will add this change in the next PR